plugins/defaultplugins/ifplugin/interface_config.go

```diff
@@ -248,6 +252,35 @@ func (plugin *InterfaceConfigurator) ConfigureVPPInterface(iface *intf.Interface
 	return wasError
 }
 
+/**
+	Set rx-mode on specified VPP interface
```

Comment:
add to the comment info about what interface types can use what modes

Reply:
done